web/content/docs/benchmarks/richards-flow/infiltration_1D.pandoc

```diff
@@ -0,0 +1,62 @@
++++
+author = "Marc Walther"
+weight = 142
+project = "Parabolic/Richards/infiltration_1D/"
+date = "2020-04-29T09:41:09+02:00"
+title = "Infiltration 1-dimensional"
+
+[menu]
+  [menu.benchmarks]
+    parent = "richards-flow"
+
++++
+
+{{< data-link >}}
+
+
+## Overview
+
+This benchmark evaluates vertical water infiltration into an initially unsaturated soil in a 1-dimensional setup. The original example is given as Tutorial 1 in the Hydrus tutorial [1] and the setup will be described in the following. We compare results from HYDRUS-1D, OpenGeoSys5 and OpenGeoSys6.
+
+
+## Problem description
+
+The problem describes a transient infiltration process. The area of the simulation domain that is evaluated is $1 m$ deep and initially unsaturated where the initial condition is defined as hydraulic pressure $p_{IC} = -1 m = -9810 Pa$. The upper boundary condition is a Dirichlet type with $p_{top} = 0.01 m = 98.1 Pa $. The original lower boundary condition in HYDRUS-1D is set as a "free drainage" type (see [2]); in both OGS versions, we extend the model domain to become $2 m$ and mimic the same behavior (the lower boundary condition for OGS is a no-flow Neumann type). The domain is discretized in HYDRUS-1D with line segments of $0.01 m$, in OGS with line segments of $0.005 m$.
+
+The parametrization is summarized in the following table. HYDRUS-1D and OGS5 use the classical Mualem-van-Genuchten formulation with parameters $\alpha$ and $n$, while OGS6 uses the formulation given in ###.
```

Comment:
` ... given in ###.` Is this correct or did you forget the reference?

Reply:
Ah, good spotting! No, I did not forget this initially (yet, when pushing), but it was intended to link to the description in doxygen. Unfortunately, this description is still missing.